Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -5713,8 +5713,8 @@ private ObservableList<ElementInfo> FindAllElementsFromPOM(string path, PomSetti
 
                         IWebElement webElement = GetWebElement(parentContext, htmlElemNode, elementTypeEnum.Item2, isShadowRootDetected);
 
-                        // Skip invisible elements
-                        if (!IsElementVisible(webElement))
+                        /// Skip invisible elements (but allow SVG elements even if they might appear invisible)
+                        if (!IsElementVisible(webElement) && !IsSvgElement(elementTypeEnum.Item2))
```

Comment:
Remove this function call and ShouldLearnElement handle the element check for SVG